### PR TITLE
got the part working that adds shit to my database!

### PR DESCRIPTION
**Pull request by marcusxavier85**

### CODE REVIEW

File: insertPokemon.php [hunk 1]
```diff
@@ -1 +1,16 @@
...
+require('functions.php');
+
+$dataBase = getDatabaseObject();
+
+$pokedexNumber = $_POST['addPokedexNo'];
+$pokemonName = $_POST['addPokemonName'];
+$pokemonType = $_POST['addPokemonType'];
+$pokemonImage = $_POST['addPokemonImage'];
+
+$insert = $dataBase->prepare('INSERT INTO `kanto_pokedex` (`pokedex_no`, `name`, `type`, `img_source`) VALUES (?, ?, ?, ?)');
```
Comment (ellie-r) [inline insertPokemon.php]: missing ; in the sql statement string

File: pokedexForm.php [hunk 2]
```diff
@@ -14,10 +21,12 @@
             Pokemon Type:
             <input type="string" name="addPokemonType" required="">
```
Comment (ellie-r) [inline pokedexForm.php]: not sure you need required="", you can just put the word required

**Review by ellie-r — CHANGES_REQUESTED** [review]
looks good just couple things to possibly change

File: pokedexForm.php (hunk 2 above)
Comment (marcusxavier85) [reply pokedexForm.php]: removed

**Review by ellie-r — APPROVED** [review]

**Pull request merged**